docs/options.md

```diff
@@ -263,3 +262,19 @@ Valid options:
 | Default      | CLI Override                                                | API Override                                                |
 | ------------ | ----------------------------------------------------------- | ----------------------------------------------------------- |
 | `"preserve"` | <code>--prose-wrap <always&#124;never&#124;preserve></code> | <code>proseWrap: "<always&#124;never&#124;preserve>"</code> |
+
+## HTML Whitespace Sensitivity
+
+_available in v1.15.0+_
```

Comment:
I wonder if these should be formatted differently? Perhaps:

> *First available in vX.X.X. Deprecated in vY.Y.Y*

Reply:
Since there's a `+`, I guess it should be fine?